spec/controllers/participant_groups_controller_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+RSpec.describe ParticipantGroupsController, type: :controller do
+
+  let(:valid_attributes) { FactoryGirl.build(:participant_group).attributes }
+  let(:invalid_attributes) { FactoryGirl.build(:participant_group, event: nil).attributes }
+
+  # This should return the minimal set of values that should be in the session
+  # in order to pass any filters (e.g. authentication) defined in
+  # ApplicationsController. Be sure to keep this updated too.
+  let(:valid_session) { {} }
+
+  before(:each) do
+    request.env['HTTP_REFERER'] = root_url
```

Comment:
`root_url` is also the path the user gets redirected to in case of a RedirectBackError, it would be better using a different path.